corehq/pillows/xform.py

```diff
@@ -201,6 +205,7 @@ def get_xform_pillow(pillow_id='xform-pillow', ucr_division=None,
         index_info=XFORM_INDEX_INFO,
         doc_prep_fn=transform_xform_for_elasticsearch,
         doc_filter_fn=xform_pillow_filter,
+        skip_doc_exists_check=skip_doc_exists_check
```

Comment:
how can this be set globally?

Reply:
This is a keyword argument to the pillow initialisation method so it can be passed to the pillow from app-process.yml https://github.com/dimagi/commcare-hq/pull/28574/files#diff-388d043ddbd6cd39a41226374948be5e3c64ee3ca86ce9053d7e0ab2d7002fd1R174